hotel/templates/emails/room_assignment.txt

```diff
@@ -1,20 +1,39 @@
 {{ room.first_names|join_and }},
 
-This email to confirm your {{ c.EVENT_NAME }} hotel staff room assignments.  Your roommates are listed below, so please take a moment to look over the information.  We will assume that all rooms are ok if you don’t reply to this email by {{ c.ONE_WEEK_OR_TAKEDOWN|datetime }}.
+This email is to confirm your upcoming {{ c.EVENT_NAME }} hotel staff room assignments.  Your roommates are listed below, so please
+take a moment to look over the information.  We will assume that all rooms are ok if you don’t reply to this email by 
+{{ c.ONE_WEEK_OR_TAKEDOWN|datetime }}.
```

Comment:
Did you meant to split this across multiple lines?  The way you have it here, there will be a line break after the word "please".  Similarly, there are line breaks below after the words "keys", "vacate", and "you".